### ISSUE TITLE
[Question] How to format code using Prettier in Gitpod ?

### ISSUE THREAD

**Issue by kevinadhiguna** [gitpod-io/gitpod#6224]
### Bug description

I installed Prettier successfully in Gitpod. Usually in VSCode in my machine, I press `ctrl + shift + P` so that a prompt appears and I can format documents using Prettier.

However, `ctrl + shift + P` opens me a new tab in Firefox. What should I do?

### Steps to reproduce

1) Install Prettier. Make sure it has been installed successfully.

After that I have not used that for long time but got confused when `ctrl + shift + P` did not work.

### Expected behavior

I can format code using Prettier.

### Anything else?

I use the latest version of Firefox browser. Any help is appreciated.

**Comment by akosyakov** [comment]
Yeah, unfortunately browsers don't allow to override all keybindings. You have several options:
- use the command pallette to trigger formatting, i.e. F1 -> Format document
- change the keybinding for Format Document
- use VS Code Desktop: https://www.gitpod.io/docs/develop/vscode-desktop-support
